packages/base-map/src/index.tsx

```diff
@@ -102,6 +105,9 @@ const BaseMap = ({
   const [hiddenLayers, setHiddenLayers] = useState(
     toggleableLayers.filter(layer => !layer?.visible).map(layer => layer.id)
   );
+  const [activeBaseLayer, setActiveBaseLayer] = useState(
+    typeof baseLayer === "object" ? baseLayer?.[0] : baseLayer
```

Comment:
From the type definitions, there is an assumption that `baseLayer` can be `null`. Is that correct? Also, `typeof null === 'object'` is true. So, a null check is probably needed on lines 142 and 154. Alternately, create a variable for whether `baseLayer` is a non-null array and use that throughout this component.

Reply:
Very good catch thank you! Forgot `null` is an object how we love javascript